### PR TITLE
Change i18n from json to yaml

### PR DESCRIPTION
**Pull request by zeina-idris**
* Changed `<i18n>` tags inside all components to yaml format, as well as changing the main translation files `en/de.json` to `en/de.yaml`

### CODE REVIEW

**Comment by zeina-idris** [comment]
Solves #96

File: src/components/useraccount/UserProfile.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
-   <div v-if="me"
-        class="my-account-content">
+  <div v-if="me"
+       class="my-account-content">
```
Comment (lauraluiz) [inline src/components/useraccount/UserProfile.vue]: 👍

File: src/components/header/LoginButton.vue [hunk 2]
```diff
@@ -67,26 +67,20 @@ export default {
...
 
 <style lang="scss">
   .list-item-user > .nav-list > .list-item-user {
     @media screen and (max-width: 767px) {
       width: 100%;
-      border: none; 
+      border: none;
     }
   }
 </style>
```
[the remark this answers is not in the document]
Comment (zeina-idris) [reply src/components/header/LoginButton.vue]: Yes, you're right. done!

**Comment by zeina-idris** [comment]
OOPS! I meant Style tag before i18n tag, excuse me @lauraluiz

**Comment by zeina-idris** [comment]
@lauraluiz  I checked if it complains about the long lines, it does but so far we don't have anything that is that long. so no complaints so far..

**Pull request merged**